site/app/controllers/admin/EmailRoomSeatingController.php

```diff
@@ -104,11 +104,12 @@ private function replacePlaceholders($message, $data) {
 
         foreach($replaces as $key => $variable) {
             if(isset($data[$key])) {
```

Comment:
You'll want to get rid of this if check around the `str_replace` as the `??` is now handling the case for when `!isset($data[$key])`.